deploy/crds/hco01.crd.yaml

```diff
@@ -18,16 +17,16 @@ spec:
         description: V2VVmware is the Schema for the v2vvmwares API
         properties:
           apiVersion:
-            description: |-
-              APIVersion defines the versioned schema of this representation
-                                      of an object. Servers should convert recognized schemas to the latest
-                                      internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources
+            description: "APIVersion defines the versioned schema of this representation\n\
```

Comment:
Why we are modifying that file (and others) if there is only one stored version in the CRD?

Reply:
right, good point